webpack.config.cjs

```diff
@@ -39,6 +40,19 @@ function extensionModules( mode, target ) {
 				test: /\.tsx?$/,
 				use: 'ts-loader',
 			},
+			{
+				// If you enable `experiments.css` or `experiments.futureDefaults`, please uncomment line below
+				// type: "javascript/auto",
```

Comment:
Could we remove this? I'm assuming it was accidentally committed, or was it deliberate?

Reply:
This was copied over from the documentation so I would say it's good to keep around?

Reply:
I see. I find the message a bit cryptic though, I had to do some googling to figure out what this was about.

> https://webpack.js.org/configuration/experiments/#experimentscss
> Enable native CSS support. Note that it's an experimental feature still under development and will be enabled by default in webpack v6

I think if we would enable this feature, we would at that point look into everything that needs to be done to make the feature work. So I'm not sure it's very useful to keep the comment around.

Reply:
If documentation had it there, it probably was meant to act as a helpful pointer. Agree that we would figure out things more if ever needed to use it, but since I have already merged the PR (because it was approved), and leaving the comment there is a trivial bit, I will let it be. Feel free to remove it, when you are working around the same lines in code next time :)

Reply:
It's ok, it's just a detail :) I had approved indeed, you did the right thing merging it 👌